crates/tx5/src/backend/go_pion.rs

```diff
@@ -0,0 +1,136 @@
+//! go pion backend
+
+use super::*;
+use crate::Config;
+
+struct GoCon(tx5_connection::FramedConn);
+
+impl BackCon for GoCon {
+    fn send(&self, data: Vec<u8>) -> BoxFuture<'_, Result<()>> {
+        Box::pin(async { self.0.send(data).await })
+    }
+
+    fn pub_key(&self) -> &PubKey {
+        self.0.pub_key()
+    }
+
+    fn is_using_webrtc(&self) -> bool {
+        self.0.is_using_webrtc()
+    }
+
+    fn get_stats(&self) -> tx5_connection::ConnStats {
+        self.0.get_stats()
+    }
+}
+
+struct GoConRecv(tx5_connection::FramedConnRecv);
+
+impl BackConRecv for GoConRecv {
+    fn recv(&mut self) -> BoxFuture<'_, Option<Vec<u8>>> {
+        Box::pin(async { self.0.recv().await })
+    }
+}
+
+struct GoWaitCon(
```

Comment:
When the inner fields aren't trivial, I'd prefer named fields

Reply:
Although I see these get extracted into named fields right away in the function below... So I guess as long as the code is written that way, you don't actually have to work too hard to work out what the `0`, `1` and `2` are

Reply:
Yeah, that's fair. It started out as one, then gradually went up to three. I should switch it to the struct form.

Reply:
done: https://github.com/holochain/tx5/pull/105/commits/681fd982b431bff9300833f093e9a472f91e3dbd